CPAC/nuisance/utils/compcor.py

```diff
@@ -132,30 +153,43 @@ def cosine_filter(
     #    Prior to release 0.12, Nipype was licensed under a BSD license.
 
     # Modifications copyright (C) 2019 - 2024  C-PAC Developers
-    from nipype.algorithms.confounds import _cosine_drift, _full_rank
```

Comment:
Does this https://github.com/FCP-INDI/C-PAC/blob/db9186285b2214bcc8a2f322d9b98fe10eac1102/CPAC/nuisance/utils/compcor.py#L173 work without these imports?

Reply:
Thanks for the review. Corrected and tested now.
the correlation test results are [here](https://github.com/birajstha/acompcor_cosine_fiter_correlation_test/blob/main/correlation.txt)